### PR TITLE
Catch panic to file logger.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-960.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -201,6 +201,25 @@ func startAutoPopUpLoop(ctx context.Context, cfg *looper.Config, period uint32,
...
+		case string:
+			if err == "fatal log event" {
+				panic(err)
+			}
+			v = err.(string)
+		default:
+			v = fmt.Sprintf("error %+v", v)
+		}
+		logger.Fatal(v)
+		panic(err)
```
Comment (furkhat) [inline main.go]: this panic is redundant. `logger.Fatal` calls panic.

Comment (furkhat) [reply main.go]: I may be wrong, but this seems like an infinite panic-recover-panic cycle. Did you test it?

Comment (dzeckelev) [reply main.go]: You're wrong. If a panic has already been processed by a logger, then the function ends on this line.
https://github.com/Privatix/dappctrl/blob/4a1c7459ceab7a9eb7c28cc23c5bbe8bbb34613c/main.go#L212
After that, the `defer bugsnag.PanicHunter()` will catch and process the panic.
Yes, I tested it.

Comment (dzeckelev) [reply main.go]: `logger.Fatal(v)`  is necessary to catch an unprocessed panic.

Comment (furkhat) [reply main.go]: why panic on the next line?

Comment (dzeckelev) [reply main.go]: @furkhat this is an extra string. Thank you. Updated.

**Pull request merged**